lib/app/modules/meetings/controllers/meetings_controller.dart

```diff
@@ -1,30 +1,38 @@
 import 'package:flutter/material.dart';
 import 'package:get/get.dart';
 
```

Comment:
where are the functions that are automatically created by get_cli like on ready function , on dispose function and others in controller file
if you have removed that make sure you dont do that from next time as it plays very important role to prevent memory leaks
just add these functions again

```
  @override
  void onInit() {
    super.onInit();
  }

  @override
  void onReady() {
    super.onReady();
  }

  @override
  void onClose() {
    super.onClose();
  }

  void increment() => count.value++;
  
```

Reply:
I haven't removed these functions. These just weren't there when I was working on the controller